### DISCUSSION TITLE
Multisite or multilang

### DISCUSSION THREAD

**Discussion by jozoXXL** [trendyminds/craft-palette#53] [General]
Do you need to do something so it works with multisite or multilang setup..

**Comment by wmdhosting** [comment]
Is there any reaason is not posibllel to find entries on others sites, it works only with main site...
@aaronbushnell

**Reply by aaronbushnell** [reply]
Getting Palette to work for each site uniquely is pretty challenging due to the way the JS is inserted and how to "detect" which site you're currently on.

I've not had time to devote to this but PRs are certainly welcome!

**Reply by wmdhosting** [reply]
OK thx for info. WIll check with some devs what can be done.. :)

if you have any pointers where to start will save some time..
